### ISSUE TITLE
请问能否只使用CPU？ 当前没有GPU 卡

### ISSUE THREAD

**Issue by mdys** [netease-youdao/EmotiVoice#54]
如题，感谢！

**Comment by syq163** [comment]
Certainly, it is feasible. Please refer to https://github.com/netease-youdao/EmotiVoice/wiki/Hardware-and-Software-Requirements-for-Running-EmotiVoice for details.
